src/pages/LandingPage/EmailConfirm.jsx

```diff
@@ -8,12 +8,12 @@ const EmailConfirm = ({email, message}) => {
         <br />
         <div className='bg-gray-800 p-6 rounded-lg'>
             <h3 className="text-xl font-bold mb-4">Here is a copy of your email: </h3>
-            <p className='py-1'>
+            <p className='my-2'>
```

Comment:
my-2 sets the padding in the Y-direction, meaning both top and bottom. In this case the requirement was 8px bottom, so then mb-2 (margin-bottom-2) should be used. 

But, both lookes fine, so its not really a problem right now. However, many designers are strict and are salty with devs that doesnt pay attention, hehe. So its something to think about!

Reply:
Okay. I will work on that.